releasenotes/notes/crawler-webdriver-di-71225500f3751983.yaml

```
@@ -0,0 +1,4 @@
+---
+features:
+  - |
+    Add optional webdriver parameter to Crawler in order to allow DI of a pre-configured webdriver instead of creating a new one in the constructor
```

```suggestion
    Add an optional webdriver parameter to Crawler in order, to allow the usage of a pre-configured webdriver instead of creating a new one in the constructor
```

I would like to simplify the description a little...

---

```suggestion
    Add an optional webdriver parameter to Crawler in order, to allow the usage of a pre-configured webdriver instead of creating a new one in the constructor.
```
Just adding a full stop to @anakin87 suggestion to keep with the release notes/documentation conventions.